syntax = "proto3";

message List1 {
    int32 number1 = 1;
    int32 number2 = 2;
}

message Num {
    int32 value = 1;
}


service Calculator {
    rpc Sum(List1) returns (Num) {}
}